plugins/scaffolder-backend/report.api.md

```diff
@@ -354,7 +354,11 @@ export const createPublishGitlabMergeRequestAction: (options: {
     sourcePath?: string | undefined;
     targetPath?: string | undefined;
     token?: string | undefined;
-    commitAction?: 'auto' | 'update' | 'delete' | 'create' | 'skip' | undefined;
+    commitAction?: 'auto' | 'update' | 'delete' | 'create' | 'skip' | undefined
+    /**
+     * @public
+     * @deprecated use import from `@backstage/plugin-scaffolder-backend-module-github` instead
```

Comment:
Wait where did this come from? :) I don't think I see it elsewhere in the code and this is an autogenerated file eh?

Reply:
Nah leave it in if that's how it got generated. Curious thing tho 🤔

Reply:
Now it wasn't added anymore by the api report generator.. weird 🤷